src/common/nodelist-utils/test/test.browser.js

```diff
@@ -0,0 +1,89 @@
+const expect = require('chai').expect;
+const NodeListUtils = require('../');
+const findNodeWithFirstChar = NodeListUtils.findNodeWithFirstChar;
+
+document.body.innerHTML = `
```

Comment:
Overwriting the document body could mess with other tests. Any changes we make to the document should be undone via the mocha before/after hooks.

In this case I don't think we actually need the dom nodes to be attached to work with them here and so we should be able to just used a detached div to build the lists, eg:

```javascript
const CONTAINER = document.createElement("div");
CONTAINER.innerHTML = `
  ....
`
```

And then use `CONTAINER.querySelector` instead of `document.querySelector` which avoids the problem entirely.

Reply:
Good call. Will fix.

Reply:
So THAT'S what caused the element scroll test to break in my local env!!